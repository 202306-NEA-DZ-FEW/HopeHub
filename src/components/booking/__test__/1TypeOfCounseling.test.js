import renderer from "react-test-renderer";

import TypeOfCounseling from "../1TypeOfCounseling";

it("renders correctly", () => {
    const tree = renderer.create(<TypeOfCounseling />).toJSON();
    expect(tree).toMatchSnapshot();
});
