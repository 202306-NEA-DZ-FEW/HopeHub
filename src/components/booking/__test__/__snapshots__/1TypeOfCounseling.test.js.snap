// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-NeutralWhite min-w-screen mb-12"
>
  <div
    className="w-full h-full px-8 lg:px-20 bg-NeutralWhite "
  >
    <div
      className="mb-3 pt-12 font-poppins font-semibold tracking-wider text-NeutralBlack capitalize text-2xl lg:text-4xl leading-normal"
    >
      Let's match you with the right therapist
    </div>
    <div
      className="font-poppins font-regular text-justify text-NeutralBlack text-base lg:text-lg leading-relaxed"
    >
      Please fill out this short questionnaire to provide some general and anonymous background about you and the issues you'd like to deal with in online therapy. It would help us match you with the most suitable therapist for you.
    </div>
    <div
      className=" flex flex-col bg-white lg:w-3/5 lg:h-1/2 sm:w-full sm:h-[80%] sm:leading-tight mx-auto mt-14 shadow-[4.0px_8.0px_8.0px_rgba(0,0,0,0.42)] rounded-lg relative"
    >
      <h3
        className="py-5 px-4 lg:py-10 lg:px-11 leading-normal text-NeutralBlack lg:text-2xl text-2xl font-regular font-poppins capitalize "
      >
        What type of counseling are you looking for?
      </h3>
      <div
        className="px-3 mx-5 mt-1 lg:px-6  group "
      >
        <h3
          className="rounded-md py-4 shadow-[rgba(0,_0,_0,_0.24)_0px_3px_8px] pl-2 lg:px-5 lg:py-5 cursor-pointer text-NeutralBlack group-hover:text-NeutralWhite group-hover:bg-Accent group-hover:scale-105 duration-300 text-lg lg:text-xl font-regular font-poppins capitalize
                            "
          onClick={[Function]}
        >
          Individual counseling
        </h3>
      </div>
      <div
        className="px-3 mx-5 lg:px-6 mt-1 lg:mt-2 group"
      >
        <h3
          className="rounded-md py-4 shadow-[rgba(0,_0,_0,_0.24)_0px_3px_8px] pl-2 lg:pl-5 lg:py-5 cursor-pointer text-NeutralBlack  group-hover:text-NeutralWhite group-hover:bg-Accent group-hover:scale-105 duration-300 text-lg lg:text-xl font-regular font-poppins capitalize
                                "
          onClick={[Function]}
        >
          Teen counseling (for my child)
        </h3>
      </div>
      <div
        className="self-end pr-6 py-10 lg:py-10 lg:pr-11 group "
      >
        <button
          className="w-28 h-10 rounded-md text-base font-poppins font-regular bg-Accent text-NeutralBlack group-hover:bg-[#879AB8] group-hover:text-NeutralWhite group-hover:scale-105 duration-500"
          onClick={[Function]}
        >
          Next
        </button>
      </div>
    </div>
  </div>
</div>
`;
